src/datatip.jl

```diff
@@ -1,46 +1,62 @@
 #=
-@FIXME?
-If we come to be able to use our full-featured components in atom-julia-client
-for datatips, we may want to append method tables again.
-Ref: https://github.com/JunoLab/atom-julia-client/blob/master/lib/runtime/datatip.js#L3-L9
+@TODO:
+Use our own UI components for this: atom-ide-ui is already deprecated, ugly, not fully functional, and and...
+Once we can come to handle links within datatips, we may want to append method tables as well
 =#
 
 handle("datatip") do data
-  @destruct [mod || "Main", word] = data
-  docs = @errs getdocs(mod, word)
+  @destruct [
+    word,
+    mod || "Main",
+    path || nothing,
+    row || 1,
+    column || 1
+  ] = data
+
+  if isdebugging() && (datatip = JunoDebugger.datatip(word, path, row, column)) !== nothing
+    return Dict(:error => false, :strings => datatip)
+  end
 
+  docs = @errs getdocs(mod, word)
   docs isa EvalError && return Dict(:error => true)
-  occursin(nobinding_regex, string(docs)) && return Dict(:novariable => true)
 
-  Dict(:error   => false,
-       :strings => makedatatip(docs))
+  # don't show verbose stuff
+  docstr = replace(string(docs), nodoc_regex => "")
+  occursin(nobinding_regex, docstr) && return Dict(:error => true)
+
+  datatip = []
+
+  val = getfield′′(getmodule′(mod), Symbol(word))
```

Comment:
maybe this line should be replaced with another better way, or needs some tweaks, I think.

Reply:
should be improved by #174